server/server_test.go

```diff
@@ -77,7 +80,132 @@ func forwardSignal(t *testing.T, outbound chan<- struct{}, inbound <-chan os.Sig
 	var s struct{}
 	sig := <-inbound
 	if sig != os.Interrupt {
-		t.Errorf("Unexpected signal: %s", sig.String())
+		t.Errorf("Unexpected signal: %s\n", sig.String())
 	}
 	outbound <- s
 }
+
+func Test_newSocketServer(t *testing.T) {
+	cfg := new(config.Configuration)
+	cfg.Socket = "chose_your_socket_addr:chose_your_socket_port"
+
+	mock_server := &http.Server{
+		Addr:         cfg.Socket,
+		Handler:      nil,
+		ReadTimeout:  15 * time.Second,
+		WriteTimeout: 15 * time.Second,
+	}
+
+	if ret := newSocketServer(cfg, nil); ret != nil {
+		switch {
+		case ret.Addr != mock_server.Addr:
+			t.Errorf("[Test_newSocketServer] Addr invalide: %v != %v\n",
+				ret.Addr, mock_server.Addr)
+			fallthrough
+		case ret.ReadTimeout != mock_server.ReadTimeout:
+			t.Errorf("[Test_newSocketServer] ReadTimeout invalide: %v != %v\n",
+				ret.ReadTimeout, mock_server.ReadTimeout)
+			fallthrough
+		case ret.WriteTimeout != mock_server.WriteTimeout:
+			t.Errorf("[Test_newSocketServer] WriteTimeout invalide: %v != %v\n",
+				ret.WriteTimeout, mock_server.WriteTimeout)
+		}
+		ret.Close()
+	} else {
+		t.Errorf("[Test_newSocketServer] ret is Nil")
+	}
+}
+
+func Test_newMainServer(t *testing.T) {
+	const (
+		chose_your_socket_port    = 8000                        //chose_your_socket_port
```

Comment:
Can you please use different variable names? Please omit "chose_your_" part of it.